.env.template

```diff
@@ -5,13 +5,19 @@
 # ╚════██║██║     ██╔══██╗██║██╔═══╝    ██║   ╚════██║ #
 # ███████║╚██████╗██║  ██║██║██║        ██║   ███████║ #
 # ╚══════╝ ╚═════╝╚═╝  ╚═╝╚═╝╚═╝        ╚═╝   ╚══════╝ #
+# Created by @westsurname                              #
+#                                                      #
+# The headings contain the scripts that require the    #
+# the variables within.                                #
 #------------------------------------------------------#
 
-#--------#
-# SERVER #
-#--------#
+#--------------#
+# Blackhole UI #
+#--------------#
 
-SERVER_DOMAIN=<server_domain>
+SERVER_DOMAIN=<server_domain> #add if using a domain name / reverse proxy
```

Comment:
Is this only pertinent to the ui?

Reply:
During development it was, but I may have changed it. It was a cors variable.

Reply:
Cors is "*" so server domain is no longer needed